yb-voyager/cmd/import.go

```diff
@@ -157,6 +158,8 @@ func registerImportSchemaFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&target.IgnoreIfExists, "ignore-exist", false,
 		"true - to ignore errors if object already exists\n"+
 			"false - throw those errors to the standard output (default false)")
+	cmd.Flags().BoolVar(&flagRefreshMViews, "refresh-mviews", false,
+		"If set, refreshes the materialised views on target after importing data (default false)")
```

Comment:
nit: "If set, refreshes the materialised views on target during post import data phase (default false)"